run_route_scripts/run_with_server.py

```diff
@@ -85,7 +85,7 @@ def make_request(post_body):
   parser.add_argument('--output-dir', type=str, help='The directory in which to place the result of each request')
   parser.add_argument('--concurrency', type=int, help='The number of processes to use to make requests', default=multiprocessing.cpu_count())
   parser.add_argument('--format', type=str, help='Supports csv, json, raw and null output formats', default='csv')
-  parser.add_argument('--headers', type=str, help='Additional http headers to send with the requests. Follows the http header spec, eg. some-header-name: some-header-value', action='append', nargs='*')
+  parser.add_argument('--headers', type=str, help='Additional http headers to send with the requests. Follows the http header spec, eg. some-header-name: some-header-value', action='append', nargs='*', default=[])
```

Comment:
this script failed if `--headers` parameter wasn't specified. adding default value fixes it

Reply:
strange! i swore i tested that... thank you for cleaning it up!

Reply:
@mandeepsandhu this is where you saw that